### PR TITLE
change: y-final from U+1540 to U+1429

### PR DESCRIPTION
**Pull request by eddieantonio**
ᑖᓂᓯ!

This is just to change the Y-final. I had also though it was U+1540, but after I did an informal survey (asking Cree writers), they almost unanimously prefered the way U+1429 CANADIAN SYLLABICS FINAL PLUS renders; in most fonts, it renders like the other finals, whereas U+1540 renders in the center of the text and is the only final to do so.

Just FYI, I've come up with a "definitive" list of syllabics for writing in ᓀᐦᐃᔭᐍᐏᐣ! https://github.com/UAlbertaALTLab/nehiyawewin-syllabics

### CODE REVIEW

**Comment by eddieantonio** [comment]
> What's the NARROW NO-BREAK SPACE used for?

It's used as a morpheme separator by a few _very_ particular writers; not everybody uses it, but those that do use it REALLY want it! It's a space that's half as wide as a normal space, but importantly, it does NOT break words! This is used in, e.g., «ᑭᑲ ᐚᐸᒥᑎᐣ» ("I will see you") to separate the gramatical morphemes «ᑭᑲ» from the rest of the word.

> I'm looking forward to working with you more!

Likewise 😄

**Comment by dwhieb** [comment]
Interesting! Do other writers use hyphens (like [this SRO passage](https://github.com/dwhieb/Plains-Cree/blob/535f5c945d0c69b2e885ff42715ca25dd7802758/test.js#L8) I got from [this Wikipedia page](https://en.wikipedia.org/wiki/Western_Cree_syllabics)), or just write them as a single word, or both?

**Comment by eddieantonio** [comment]
tl;dr: both!

Some people use hyphens in SRO, as dictated in [How to Spell it in Cree](https://creeliteracy.org/how-to-spell-it-in-cree-wolvengrey-okemasis/). It helps for writing long words with multiple preverbs like [ninôhtênêhiyawi-kiskinwahamâkosin](https://sapir.artsrn.ualberta.ca/cree-dictionary/search?q=nin%C3%B4ht%C3%AAn%C3%AAhiyawi-kiskinwaham%C3%A2kosin).

However, some writers skip the hyphens, especially in shorter words like ê-wâpamât/ewâpamât. Additionally, the diacritic on the ê is omitted by some writers, since there is no such \*e grapheme in "strict" SRO!
